attribute-masking-service/src/main/java/uk/gov/gchq/palisade/service/attributemask/stream/SerDesConfig.java

```diff
@@ -120,6 +120,15 @@ private SerDesConfig() {
         };
     }
 
+    /**
+     * Kafka value serialiser for downstream messages going out as output
+     *
+     * @return an appropriate value serialiser for the topic's message content (AttributeMaskingResponse)
+     */
+    public static Serializer<byte[]> passthroughValueSerializer() {
```

Comment:
Is there no standard `ByteSerializer` provided in dependencies?

Reply:
this is just a lambda fn. provided only for clarity in the code